DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeIDGenerator.swift

```diff
@@ -63,21 +68,4 @@ internal final class NodeIDGenerator {
         return nextID
     }
 }
-
-// MARK: - UIView tagging
-
-fileprivate var associatedNodeIDKey: UInt8 = 1
-fileprivate var associatedNodeIDsKey: UInt8 = 2
-
-private extension UIView {
-    var nodeID: NodeID? {
-        set { objc_setAssociatedObject(self, &associatedNodeIDKey, newValue, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC) }
-        get { objc_getAssociatedObject(self, &associatedNodeIDKey) as? NodeID }
-    }
-
-    var nodeIDs: [NodeID]? {
-        set { objc_setAssociatedObject(self, &associatedNodeIDsKey, newValue, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC) }
-        get { objc_getAssociatedObject(self, &associatedNodeIDsKey) as? [NodeID] }
-    }
-}
```

Comment:
Why this removal?

Reply:
Since now key is derived from view and recorder it made more sense to me to keep dictionary instead of using associated object.